engine/install/linux-postinstall.md

```diff
@@ -191,11 +191,11 @@ Configuring Docker to accept remote connections can be done with the `docker.ser
     $ sudo systemctl restart docker.service
     ```
 
-6.  Check to see whether the change was honored by reviewing the output of `netstat` to confirm `dockerd` is listening on the configured port.
+6.  Check to see whether the change was honored by reviewing the output of `ss` to confirm `dockerd` is listening on the configured port.
```

Comment:
How common is it to have `ss` installed? I understand it mostly replaces `netstat`, but `netstat` (I think) is more common to have installed. Wondering if we'd need to have a note which package provides it (but not sure it's the same package name between RPM-based and Debian-like systems)

Reply:
As decribed here:
https://bugzilla.redhat.com/show_bug.cgi?id=1119297
And here:
https://github.com/inspec/inspec/issues/1655

We can found most linux distribution going to iproute rather thatn net-tools.

Reply:
I have to admit I also wasn't familiar with `ss`, but it is installed (both on my host, and in my `network-toolbox` container), so I'll probably start adjusting my own usage. :sweat_smile:

For the benefit of our users though, we should probably add an extra note here, something like:

```suggestion
6.  Check to see whether the change was honored by reviewing the output of `ss` (or `netstat`) to confirm `dockerd` is listening on the configured port.
```